src/infinite-grid.component.ts

```diff
@@ -54,12 +54,17 @@ export class CoolInfiniteGridComponent implements OnInit, OnDestroy {
     private moveTopBoundary: number;
     private moveBottomBoundary: number;
 
-    private scrollHandler: EventListener;
-    private resizeHandler: EventListener;
-
     @ContentChild(TemplateRef)
     private template: TemplateRef<Object>;
 
+    @HostListener('window:scroll') scrollHandler() {
```

Comment:
please add visibility modifier and put the annotation into a separate row as with other annotations